projects/batfish-common-protocol/src/org/batfish/datamodel/Interface.java

```diff
@@ -950,9 +996,8 @@ public JSONObject toJSONObject() throws JSONException {
       JSONObject iface = new JSONObject();
       iface.put("node", _owner.getName());
       iface.put("name", _key);
-      iface.put("prefix", _prefix.toString());
```

Comment:
The change here is to refactor duplicated "type" and "prefix" strings used elsewhere in the code (existing, unmaintainable hackery :-)). JSON object serialization is used in unit test result references otherwise.